.travis.yml

```diff
@@ -11,7 +11,7 @@ matrix:
 
 script:
   - ./config/travis/run-checks.sh
-  - npm run lint frontend/src/**/*.js
+  - npm run lint frontend/src/static/*.js
```

Comment:
Why this change?

Reply:
The ```build.js``` file is auto-generated by browersify and does not conform to the eslint rules so I have excluded it here.